common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -133,8 +137,12 @@ define('video/04_video_full_screen.js', [], function () {
         fullScreenClassNameEl.removeClass('video-fullscreen');
         $(window).scrollTop(this.scrollPos);
         this.videoFullScreen.fullScreenEl
-            .attr('title', gettext('Fill browser'))
-            .text(gettext('Fill browser'));
+            .attr('aria-pressed', 'false')
+            .find('.icon')
+                .removeClass('fa-compress')
+                .addClass('fa-arrows-alt')
+            .find('.control-text')
```

Comment:
Nit: the indentation is a little confusing here. I think this find will be executed beneath ``.icon``, so as written it should probably be indented one level in. I think it would be less confusing to break this into multiple statements so it is clear the context in which the actions are being executed.